sampleWorkspace/configFileTypes/jsConfigWithESM/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "test",
+  "type": "commonjs"
```

Comment:
Deliberately specifying the opposite here - the config file is `.js` with ESM contents, the package here has `type: commonjs`, we still want it to work.

The E2E tests will try all possible combinations on top.